### PR TITLE
WIP: Minor cleanup

### PR DESCRIPTION
**Pull request by a1346054**

### CODE REVIEW

File: COPYING [hunk 1]
```diff
@@ -277,4 +277,63 @@ YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER
 PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE
 POSSIBILITY OF SUCH DAMAGES.
 
-		     END OF TERMS AND CONDITIONS
+                     END OF TERMS AND CONDITIONS
+
+            How to Apply These Terms to Your New Programs
```
Comment (rusty-snake) [inline COPYING:282]: Not needed IMHO

File: mkdeb.sh.in [hunk 2]
```diff
@@ -22,7 +22,7 @@ if [ -n "$HAVE_SELINUX" ]; then
     CONFIG_ARGS="$CONFIG_ARGS --enable-selinux"
 fi
 
-TOP=`pwd`
+TOP=$(pwd)
```
Comment (rusty-snake) [inline mkdeb.sh.in:25]: Just an idea, maybe with a fall-back substitution.
```suggestion
TOP=$PWD
```

File: COPYING (hunk 1 above)
Comment (reinerh) [reply COPYING:282]: I agree it's not needed, as it's not part of the license terms.
But I also don't mind having it in there to have a "pristine" copy of the GPL file.

**Review by reinerh — APPROVED** [review]

File: COPYING (hunk 1 above)
Comment (a1346054) [reply COPYING:282]: https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLOmitPreamble explains why it is needed

**Review by kmk3 — CHANGES_REQUESTED** [review]
> Minor cleanup

I wouldn't exactly call this minor; there are many unrelated changes in this
PR.

Some commits seem alright on their own, others I'm not so sure.

Anyway, I'd just like to leave a "please don't merge" in here for now until I
can review it properly.

**Comment by a1346054** [comment]
Rebased to fix merge conflicts.

**Review by kmk3 — CHANGES_REQUESTED** [review]
Sorry for the delay.

> * use license file from gnu.org

LGTM; bonus points for catching a violation of the GPL license itself (I hadn't
seen this one before).

> * fix missing newlines at end of files

LGTM.

I'd personally avoid doing this until there is a commit hook/CI check to detect
such things, but it does not change too many files, so sure.

By the way, what commands have you used to detect/fix this?

> * fix spelling

Please either wait for the merge of #4502 (which conflicts with this) so that
it can be a clean revert or make a separate PR for this.

As for the commit itself, there are some leftovers:

```console
$ git grep temporarely
configure:# overlayfs features temporarely disabled pending fixes
$ git grep parrent
src/firejail/usage.c:   "\tparrent interfaces.\n"
```

I have noticed the "parrent" one myself as well on #4502, though it was because
of the above file rather than the zsh completion.

Other than that, LGTM.

> * fix shellcheck warnings

NAK; there are still many shellcheck warnings if you run it locally, including
for some common pitfalls, such as the lack of quotes.

Please make a separate PR for this.

> * trim excess whitespace

I have some questions and I'd like to discuss this one separately.

Please make a separate PR for this.

**Comment by a1346054** [comment]
Thank you for the review, I will rework the PRs.

File: src/bash_completion/firejail.bash_completion.in [hunk 3]
```diff
@@ -91,10 +91,10 @@ _firejail()
             return 0
             ;;
          --net)
-         	comps=$(__interfaces)
+            comps=$(__interfaces)
```
[the remark this answers is not in the document]
Comment (a1346054) [reply src/bash_completion/firejail.bash_completion.in:94]: Fixed in new PR.

**Comment by rusty-snake** [comment]
@a1346054 can we close this PR?

**Comment by a1346054** [comment]
I reworked this PR and split it into individual PRs for further discussion, as suggested in the review comments.

**Pull request closed without merging**